app/services/battle_service.rb

```diff
@@ -292,6 +327,10 @@ def validate_player_in_battle(battle)
     raise BattleValidationError.new "Player #{@player.name} is not in battle #{battle.id}" unless battle.battle_players.find_by(player: @player).present?
```

Comment:
```suggestion
    raise BattleValidationError.new "Player #{Player.find(player_id).name} is not in battle #{battle.id}" unless battle.battle_players.find_by(player_id: player_id).present?
```

Reply:
doesn't this mean I'd need to change everywhere we call this check?

Reply:
Shouldn't be that many places in this service